conf/locale/locale_en-US.ini

```diff
@@ -595,6 +595,9 @@ settings.transfer_desc = Transfer this repository to another user or to an organ
 settings.transfer_notices_1 = - You will lose access if new owner is a individual user.
 settings.transfer_notices_2 = - You will conserve access if new owner is an organization and if you're one of the owners.
 settings.transfer_form_title = Please enter following information to confirm your operation:
+settings.wiki-delete = Erase Wiki Data
+settings.wiki-delete_desc = Once you delete a wiki, there is no going back. Please be certain.
```

Comment:
I can change to `Once you erase wiki data there is no going back. Please be certain` `erased` is past tense.

@Unknwon Sound good?